app/authentication/token.py

```diff
@@ -93,19 +88,34 @@ async def is_app_admin(token: str, app: str, error_str: str) -> bool:
         return True
 
 
-async def get_user(token: str = Depends(oauth2_scheme)) -> Tuple[str, str]:
-    """Calls GFW API to authorize user.
-
-    This functions check is user of any level is associated with the GFW
-    app and returns the user ID
-    """
+async def get_user(token: str = Depends(oauth2_scheme)) -> User:
+    """Get the details for authenticated user."""
 
     response: Response = await who_am_i(token)
 
-    if response.status_code == 401 or not (
-        "gfw" in response.json()["extraUserData"]["apps"]
-    ):
+    if response.status_code == 401:
```

Comment:
Note that there are some existing callers of `get_user`  such as `create/get/delete_api_key` and you've relaxed the authentication on them, because you are no longer checking that gfw is in their list of apps.  Is that OK?

Might be good to note somewhere if that was intentional.

Reply:
Good point. My understanding of our discussion the other day is that we think it's ok for any RW admin to also be an admin for Data API, and it shouldn't be tied to the "gfw" app anymore. I'll put in the "gfw" check back in though so we can merge this PR, we can discuss in huddle maybe and add a separate PR.

Reply:
Oh wait, I confused this with the admin check, didn't realizer we also don't let users even create an API key without the gfw app. That seems pretty unnecessary to me, especially since there's already non-GFW users. I think any user should be able to create and manage API keys

Reply:
I'm fine with that.  I don't think you have to wait for the huddle on this - just maybe confirm directly with Solomon that is fine to remove the gfw app check here.